packages/gestalt/src/TableHeaderCell.js

```diff
@@ -4,28 +4,49 @@ import cx from 'classnames';
 import styles from './Table.css';
 
 type Props = {|
+  /**
+   * The content of the table cell.
+   */
   children: Node,
+  /**
+   * Private prop required for sticky columns
```

Comment:
can these be moved to the end of the list, and are you making a follow-up PR to add `_` prefixes?